test/lib/utils/exit-handler.js

```diff
@@ -584,47 +576,48 @@ t.test('defaults to log error msg if stack is missing when unloaded', async (t)
   await exitHandler(err('Error with no stack', { code: 'ENOSTACK', errno: 127 }, true))
   t.equal(process.exitCode, 127)
   t.same(errors, ['Error with no stack'], 'should use error msg')
-  t.match(logs.error, [
-    ['code', 'ENOSTACK'],
-    ['errno', 127],
-  ])
+  t.strictSame(logs.error, [])
```

Comment:
Where did this error go?

Reply:
It will never be shown to the user if npm is not loaded. Another case of old tests not being real.